tests/php.ini-win

```diff
@@ -0,0 +1,3 @@
+[PHP]
+extension_dir = "./ext"
+extension=php_pdo_sqlite.dll
```

Comment:
no, tak, že nové nette má jako defaultní cache sqlite storage a když chci pustit testy na windows, tak se to bez tohoto neobejde. Mám to vyhodit?